megazords/ios-rust/src/lib.rs

```diff
@@ -2,22 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#![allow(unknown_lints)]
-#![warn(rust_2018_idioms)]
-
-pub use as_ohttp_client;
-pub use autofill;
-pub use crashtest;
-pub use error_support;
-pub use fxa_client;
-pub use logins;
-pub use nimbus;
-pub use places;
-pub use push;
-pub use remote_settings;
-pub use rust_log_forwarder;
-pub use suggest;
-pub use sync15;
-pub use sync_manager;
-pub use tabs;
-pub use viaduct_reqwest;
+// No code needed for this crate, it only exists to bundle all dependencies together.
+//
+// https://mozilla.github.io/application-services/book/design/megazords.html
```

Comment:
Hmm, this might not be true.  I'm going to undo this change and see if it makes CI pass.

Reply:
Yup, this was a lie.  I updated the tool to also update this file and added a comment with my guess as to what's going on.